Source/WebCore/Modules/compression/CompressionStreamEncoder.cpp

```diff
@@ -26,21 +26,125 @@
 #include "CompressionStreamEncoder.h"
 
 #include "BufferSource.h"
+#include "Exception.h"
 #include <JavaScriptCore/GenericTypedArrayViewInlines.h>
 #include <JavaScriptCore/JSCInlines.h>
 #include <JavaScriptCore/JSGenericTypedArrayViewInlines.h>
 
 namespace WebCore {
 
-RefPtr<Uint8Array> CompressionStreamEncoder::encode(const BufferSource&& input)
+ExceptionOr<RefPtr<Uint8Array>> CompressionStreamEncoder::encode(const BufferSource&& input)
 {
-    UNUSED_PARAM(input);
-    return nullptr;
+    size_t length = input.length();
```

Comment:
I'm not sure this declaration is useful unless 'input.length()' doesn't return `size_t` for some reason.

Reply:
Nice catch. Was a left over from a past iteration of the code.